diff --git a/node_modules/@capacitor-community/media/Package.swift b/node_modules/@capacitor-community/media/Package.swift
new file mode 100644
index 0000000..b3e9c87
--- /dev/null
+++ b/node_modules/@capacitor-community/media/Package.swift
@@ -0,0 +1,29 @@
+// swift-tools-version: 5.9
+import PackageDescription
+
+let package = Package(
+    name: "CapacitorCommunityMedia",
+    platforms: [.iOS(.v14)],
+    products: [
+        .library(
+            name: "CapacitorCommunityMedia",
+            targets: ["MediaPlugin"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", exact: "8.0.0"),
+        .package(url: "https://github.com/SDWebImage/SDWebImage.git", from: "5.18.0")
+    ],
+    targets: [
+        .target(
+            name: "MediaPlugin",
+            dependencies: [
+                .product(name: "Capacitor", package: "capacitor-swift-pm"),
+                .product(name: "Cordova", package: "capacitor-swift-pm"),
+                .product(name: "SDWebImage", package: "SDWebImage")
+            ],
+            path: "ios/Plugin",
+            exclude: ["MediaPlugin.m", "MediaPlugin.h", "Info.plist"],
+            sources: ["MediaPlugin.swift"]
+        )
+    ]
+)
